contrib/nginx/template.py

```diff
@@ -0,0 +1,101 @@
+pkgname = "nginx"
+pkgver = "1.24.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/etc/nginx",
+    "--user=_nginx",
+    "--group=www-data",
+    "--with-file-aio",
+    "--conf-path=/etc/nginx/nginx.conf",
+    "--error-log-path=/var/log/nginx/error.log",
+    "--http-log-path=/var/log/nginx/access.log",
+    "--lock-path=/run/nginx/nginx.lock",
+    "--modules-path=/usr/libexec/nginx",
+    "--pid-path=/run/nginx/nginx.pid",
+    "--sbin-path=/usr/bin/nginx",
+    "--http-client-body-temp-path=/var/tmp/nginx/client_body_temp",
+    "--http-fastcgi-temp-path=/var/tmp/nginx/fastcgi_temp",
+    "--http-proxy-temp-path=/var/tmp/nginx/proxy_temp",
+    "--http-scgi-temp-path=/var/tmp/nginx/scgi_temp",
+    "--http-uwsgi-temp-path=/var/tmp/nginx/uwsgi_temp",
+    "--with-compat",
+    "--with-http_addition_module",
+    "--with-http_auth_request_module",
+    "--with-http_dav_module",
+    "--with-http_flv_module",
+    "--with-http_gunzip_module",
+    "--with-http_gzip_static_module",
+    "--with-http_mp4_module",
+    "--with-http_random_index_module",
+    "--with-http_realip_module",
+    "--with-http_secure_link_module",
+    "--with-http_slice_module",
+    "--with-http_ssl_module",
+    "--with-http_stub_status_module",
+    "--with-http_sub_module",
+    "--with-http_v2_module",
+    "--with-mail=dynamic",
+    "--with-mail_ssl_module",
+    "--with-pcre",
+    "--with-stream=dynamic",
+    "--with-stream_realip_module",
+    "--with-stream_ssl_module",
+    "--with-stream_ssl_preread_module",
+    "--with-threads",
+    "--without-mail_imap_module",
+    "--without-mail_pop3_module",
+    "--without-mail_smtp_module",
+]
+make_dir = "."
+hostmakedepends = ["linux-headers"]
+makedepends = ["openssl-devel", "pcre2-devel", "zlib-devel"]
+checkdepends = [
+    "ffmpeg",
+    "libgd-devel",
+    "libgd-progs",
+    "libxslt-devel",
```

Comment:
-devel in makedepends

also sort the list

and, i think ffmpeg specifically can be left out since that doesn't matter very much  

also, libgd/libxslt have modules in nginx (disabled). perhaps maybe want to enable those too?